test/PushNotificationTests/PushNotificationTests.vcxproj.filters

```diff
@@ -13,6 +13,9 @@
       <UniqueIdentifier>{67DA6AB6-F800-4c08-8B7A-83BB121AAD01}</UniqueIdentifier>
       <Extensions>rc;ico;cur;bmp;dlg;rc2;rct;bin;rgs;gif;jpg;jpeg;jpe;resx;tiff;tif;png;wav;mfcribbon-ms</Extensions>
     </Filter>
+    <Filter Include="CopyFiles">
+      <UniqueIdentifier>{a61a0e17-91e0-4d34-9aaa-a6b871593591}</UniqueIdentifier>
```

Comment:
What is this for?

Reply:
CopyFiles fiter in the solution. Just to group the extra files that needs to be copied to the output dir together